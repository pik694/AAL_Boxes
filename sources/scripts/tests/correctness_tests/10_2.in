10
5 24 46
7 55 3
86 52 4
30 73 51
10 23 40
23 70 41
56 19 85
36 86 54
54 90 51
32 3 37

