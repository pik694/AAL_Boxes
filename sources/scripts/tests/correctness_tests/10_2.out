1 9 2 8 0 5 3 7 4 6 
720499->506639
